proto/frostdb/storage/v1alpha1/storage.proto

```diff
@@ -0,0 +1,299 @@
+syntax = "proto3";
+
+package frostdb.storage.v1alpha1;
+
+// FrostDBService is a service that allows clients to execute queries as logicalplan
+// against an instance of FrostDB.
+service FrostDBService {
+  // Query executes the query plan contained in the request and returns back
+  // the results as arrow records in IPC format.
+  rpc Query(QueryRequest) returns (QueryResponse) {}
+}
+
+// QueryRequest is the message sent to the Query gRPC endpoint.
+message QueryRequest {
+  // PlanRoot is the root of the query plan (i.e. a scan node).
+  PlanNode plan_root = 1;
+}
```

Comment:
It's pretty hard to debug individual queries with three different ones already. 
What do you think about adding this? 

```suggestion
message QueryRequest {
  // PlanRoot is the root of the query plan (i.e. a scan node).
  PlanNode plan_root = 1;
  // optional description of the query to make debugging easier
  string description = 2;
}
```

Reply:
I like the idea, just wondering how we will populate this?
This should be dynamically right?